tasks/libs/common/junit_upload_core.py

```diff
@@ -224,11 +223,13 @@ def upload_junitxmls(team_dir: Path):
         processes.append(Popen(DATADOG_CI_COMMAND + args, bufsize=-1, env=process_env, stdout=PIPE, stderr=PIPE))
 
     for process in processes:
-        _, stderr = process.communicate()
+        stdout, stderr = process.communicate()
         print(f" Uploaded {len(tuple(team_dir.iterdir()))} files for {team_dir.name}")
+        print(stdout)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Maybe it is clearer to print this before the "Uploaded..." print ?